public/doc/openapi-3-v1.2.json

```diff
@@ -1519,6 +1559,30 @@
                     }
                 }
             },
+            "WorkflowSequenceIncrement": {
+                "description": "The desired increment corresponds to current sequence of a workflow",
+                "type": "object",
+                "properties": {
+                    "increment": {
+                        "oneOf": [
```

Comment:
@hsong-rh The "one of" has challenges when generating a client and the generated code might not work.
Its like function overloading from c++
Where the same function name can take different arguments and each one would be treated differently.

In your case we might be better off passing in an object and delaying the validation till the end on the server side

{
   location: string "top|bottom|middle", #Middle would be default
  sequence: integer
}

And in the backend server code you would evaluate which code path to use based on what was passed in and if none of the paths are applicable raise an Invalid Parameter error.

There is support in openapi schema for one of but the challenge is in generating the code that is going to work across all languages. It might be pretty trivial to add it for languages that support function overloading but for ones that don't support it is difficult. You can have simple data types and complex data types. Basic types like string, integer might work but when you have complex types built on other complex types it becomes difficult.

Reply:
I tested the client for ruby and java. Ruby client works fine. Java client references to a user type but does not define it. Evidently it is a bug.

Reply:
@bzwei Once you generate the client look in the docs folder and look for a file called WorkflowSequenceIncrement.md

I have created a gist for that object
https://gist.github.com/mkanoor/84650c330b4bd73f2dc9ad92b221e97a
If you use that example it fails because there is no definition for OneOfstringinteger anywhere in the generated client.

Reply:
yes. i got the same result.